lndr-backend/src/Lndr/Handler/Credit.hs

```diff
@@ -178,6 +178,14 @@ calculateSettlementCreditRecord config cr@(CreditRecord _ _ amount _ _ _ _ _ uca
             Just "NOK" -> nok prices * priceAdjustmentForCents
             Just "SEK" -> sek prices * priceAdjustmentForCents
             Just "NZD" -> nzd prices * priceAdjustmentForCents
+            Just "IDR" -> idr prices
+            Just "MYR" -> myr prices * priceAdjustmentForCents
+            Just "SGD" -> sgd prices * priceAdjustmentForCents
+            Just "THB" -> thb prices * priceAdjustmentForCents
+            Just "VND" -> vnd prices
```

Comment:
🤔 Why do `IDR` and `VND` not have a price adjustment for cents? I'm seeing that JPY and KRW are the same way, so there **is** a precedent. It seems odd for the denominations to be inconsistent like this. What is the rationale?